lib/latextools.coffee

```diff
@@ -283,6 +297,14 @@ module.exports = Latextools =
 
         # add more here?
 
+    # if using cwl-completion, load autocomplete provider
+    if atom.config.get("latextools.commandCompletion") != 'never'
```

Comment:
Small thing, but 300--302 should probably go in `getProvider()` that way, if someone has disabled the autocompletions plugin altogether, the cwl code can be completely ignored.